Core/clim-core/gadgets/concrete.lisp

```diff
@@ -1879,47 +742,54 @@ and must never be nil.")
+                                           (- middle slider-button-half-long-dim)
+                                           (- position slider-button-half-short-dim)
                                            +black+
-                                           (+ middle slider-button-half-long-dim) (+ position slider-button-half-short-dim))
+                                           (+ middle slider-button-half-long-dim)
+                                           (+ position slider-button-half-short-dim))
                         (when (gadget-show-value-p pane)
                           (draw-text* pane (format-value (gadget-value pane)
                                                          (slider-decimal-places pane))
                                       5 ;(- middle slider-button-half-short-dim)
                                       10))) ;(- position slider-button-half-long-dim)
-            ; horizontal case           ;
+                                        ; horizontal case           ;
             (let ((middle (round (- y2 y1) 2)))
               (draw-line* pane
                           (+ x1 slider-button-half-short-dim) middle
                           (- x2 slider-button-half-short-dim) middle
                           :ink +black+)
               (draw-rectangle* pane
-                               (- position slider-button-half-short-dim) (- middle slider-button-half-long-dim)
-                               (+ position slider-button-half-short-dim) (+ middle slider-button-half-long-dim)
+                               (- position slider-button-half-short-dim)
+                               (- middle slider-button-half-long-dim)
+                               (+ position slider-button-half-short-dim)
+                               (+ middle slider-button-half-long-dim)
                                :ink +gray85+ :filled t)
               (draw-edges-lines* pane
                                  +white+
-                                 (- position slider-button-half-short-dim) (- middle slider-button-half-long-dim)
+                                 (- position slider-button-half-short-dim)
+                                 (- middle slider-button-half-long-dim)
                                  +black+
-                                 (+ position slider-button-half-short-dim) (+ middle slider-button-half-long-dim))
+                                 (+ position slider-button-half-short-dim)
+                                 (+ middle slider-button-half-long-dim))
               (when (gadget-show-value-p pane)
                 (draw-text* pane (format-value (gadget-value pane)
                                                (slider-decimal-places pane))
                             5 ;(- position slider-button-half-short-dim)
                             (- middle slider-button-half-long-dim)))))))))
-|#
 
```

Comment:
End-of-comment was removed?

Reply:
it was changed to #+(or) for the whole form.